packages/client/components/AvatarList.tsx

```diff
@@ -0,0 +1,87 @@
+import styled from '@emotion/styled'
+import graphql from 'babel-plugin-relay/macro'
+import React, {ReactElement, useRef} from 'react'
+import {createFragmentContainer} from 'react-relay'
+import useOverflowAvatars from '../hooks/useOverflowAvatars'
+import {AvatarList_users} from '../__generated__/AvatarList_users.graphql'
+import AvatarListUser from './AvatarListUser'
+import OverflowAvatar from './OverflowAvatar'
+
+const Wrapper = styled('div')({
+  alignItems: 'center',
+  display: 'flex',
+  position: 'relative',
+  width: '100%',
+  height: 32
+})
+
+const widthToOverlap = {
```

Comment:
not sure if the overlap should change as avatar size changes? If not, we can remove this